edxval/management/commands/README.md

```diff
@@ -0,0 +1,8 @@
+### Description
+
+verify_pact.py is the management command for the provider pact verification. It will be used to verify the pacts for which edx-val is the provider.
+Since pact provider verification requires a provider url where the interactions will be made, the server should be running before 
+executing the management command. Therefore,
+
+1. Run ``python manage.py runserver --settings=edxval.settings.test``. One time migrate command might be needed.
```

Comment:
Please add migrate command with flags as step 1. Quite possible that a user like me run migrate without flag.